app/Models/Transaction.php

```diff
@@ -94,12 +101,12 @@ final class Transaction extends Model
      * @var array<string, string>
      */
     protected $casts = [
-        'amount'       => BigInteger::class,
-        'gas_price'    => BigInteger::class,
-        'gas_limit'    => BigInteger::class,
-        'timestamp'    => UnixSeconds::class,
-        'sequence'     => 'int',
-        'block_height' => 'int',
+        'value'                 => BigInteger::class,
+        'gas_price'             => BigInteger::class,
+        'gas_limit'             => BigInteger::class,
+        'timestamp'             => UnixSeconds::class,
+        'transaction_index'     => 'int',
+        'block_number'          => 'int',
```

Comment:
```suggestion
        'value'             => BigInteger::class,
        'gas_price'         => BigInteger::class,
        'gas_limit'         => BigInteger::class,
        'timestamp'         => UnixSeconds::class,
        'transaction_index' => 'int',
        'block_number'      => 'int',
```

dunno if it's your code fixer but it adds extra indentation :sadmici: